packages/vuepress-theme-odyssey/styles/components/_DocsCard.scss

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018-Present Okta, Inc.
```

Comment:
`2020-Present` for all legal headers please.

Reply:
Done